ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -158,10 +176,28 @@ const FamiliesIndividuals = React.memo(({ canEdit, hasCaseReview, familyCounts,
 
   return (
     <DetailSection
-      title={`${Object.keys(familyCounts).length} Families, ${individualsCount} Individuals${title || ''}`}
+      title={(
+        <span>
+          {`${Object.keys(familyCounts).length} Families${title || ''},`}
+          <br />
+          {`${individualsCount} Individuals${title || ''}`}
+        </span>
+      )}
       content={
-        sortBy(Object.keys(familySizeHistogram)).map(size => (
-          <div key={size}>{`${familySizeHistogram[size]} ${FAMILY_SIZE_LABELS[size](familySizeHistogram[size] > 1)}`}</div>
+        sortBy(Object.entries(familySizeHistogram)).map(([size, { total, withParents }]) => (
+          <div key={size}>
+            {`${total} famil${total === 1 ? 'y' : 'ies'} with ${FAMILY_SIZE_LABELS[size] || size} individual${size === '1' ? '' : 's'}`}
```

Comment:
You may not want to display this line when the `size` is bigger than 2.

Reply:
We do want it